README.md

```diff
@@ -243,8 +259,11 @@ You can also find other help & tutorial material on the internet, like [this cla
 
 ## Asset Catalog
 
-```sh
-swiftgen xcassets -t swift4 /dir/to/search/for/imageset/assets
+```yaml
+xcassets:
+  inputs: /dir/to/search/for/imageset/assets
+  outputs:
+    templateName: swift4
```

Comment:
`output:` key missing here and in all the following snippets. (`outputs.output` is a mandatory key, right? Same as `inputs`?)

Reply:
Had to check real quick, but yeah, it's mandatory.